solr/test-framework/src/java/org/apache/solr/util/SolrJettyTestRule.java

```diff
@@ -75,9 +70,22 @@ protected void after() {
       }
       client = null;
     }
+
+    for (SolrClient solrClient : clients.values()) {
```

Comment:
The sequence of actions in a lifecycle method (and this is one) can sometimes matter.  Do things in reverse order in a closing/after type of method.  Thus clients should be closed and *then* close Jetty.  It's the reverse order from starting.